find_xml_file.rb

```diff
@@ -0,0 +1,15 @@
+
+def find_xml_file(folder)
+  files = Dir.entries(folder)
+
+  xml_file = nil
+
+  files.each do |filename|
+    if filename.match "\.xml$"
+      xml_file = filename
```

Comment:
Are you only looking for the last xml file? This will replace the xml file each time it finds one